tools/visual-pipeline-and-platform-evaluation-tool/app.py

```diff
@@ -592,6 +624,90 @@ def on_run(
+                        os.remove(video_output_path)
+
+                    param_grid = {
+                          "object_detection_device": object_detection_device.split(", "),
+                    }
+
+                    constants = {
+                        "VIDEO_PATH": input_video_player,
+                        "VIDEO_OUTPUT_PATH": video_output_path,
+                    }
+
+                    MODELS_PATH = "/home/dlstreamer/vippet/models"
+
+                    match object_detection_model:
+                        case "SSDLite MobileNet V2":
+                            constants["OBJECT_DETECTION_MODEL_PATH"] = (
+                                f"{MODELS_PATH}/public/ssdlite_mobilenet_v2_INT8/FP16-INT8/ssdlite_mobilenet_v2.xml"
+                            )
+                            constants["OBJECT_DETECTION_MODEL_PROC"] = (
+                                f"{MODELS_PATH}/public/ssdlite_mobilenet_v2_INT8/ssdlite_mobilenet_v2.json"
+                            )
+                        case "YOLO v5m":
+                            constants["OBJECT_DETECTION_MODEL_PATH"] = (
+                                f"{MODELS_PATH}/public/yolov5m-416_INT8/FP16-INT8/yolov5m-416_INT8.xml"
+                            )
+                            constants["OBJECT_DETECTION_MODEL_PROC"] = (
+                                f"{MODELS_PATH}/public/yolov5m-416_INT8/yolo-v5.json"
+                            )
+                        case "YOLO v5s":
+                            constants["OBJECT_DETECTION_MODEL_PATH"] = (
+                                f"{MODELS_PATH}/public/yolov5s-416_INT8/FP16-INT8/yolov5s.xml"
+                            )
+                            constants["OBJECT_DETECTION_MODEL_PROC"] = (
+                                f"{MODELS_PATH}/public/yolov5s-416_INT8/yolo-v5.json"
+                            )
+                        case "Person Vehicle Bike Detection":
+                            constants["OBJECT_DETECTION_MODEL_PATH"] = (
+                                f"{MODELS_PATH}/intel/person-vehicle-bike-detection-2004/FP16-INT8/person-vehicle-bike-detection-2004.xml"
+                            )
+                            constants["OBJECT_DETECTION_MODEL_PROC"] = (
+                                f"{MODELS_PATH}/intel/person-vehicle-bike-detection-2004/person-vehicle-bike-detection-2004.json"
+                            )
+                        case _:
+                            raise ValueError("Unrecognized Object Detection Model")
+
+                    # Validate channels
+                    if recording_channels + inferencing_channels == 0:
+                        raise gr.Error(
+                            "Please select at least one channel for recording or inferencing.",
+                            duration=10,
+                        )
+
+                    # Initialize the benchmark class
+                    bm = benchmark(

```

Comment:
class name should start with a capital letter (otherwise it looks like a method)